### ISSUE TITLE
World Crash when adding certain components

### ISSUE THREAD

**Issue by dotuser** [Frooxius/NeosPublic#1562]
## Describe the bug
In own world attempting to add new component similar to "ValueCopy" or any other which creates a panel with the type selection list (not including the ones where only typed input exists) results in a blank panel.

When another user attempts to add one of said components the world crashes entirely.

## Relevant issues
This issue is new to version 2021.1.3.612 and was not present prior. No other reports appear to have been made.

## To Reproduce
1. Create or edit an item
2. Add component of type "ValueCopy" under Transform>Drivers>ValueCopy (Also happened with BooleanValueDriver)
3. Depending on if host or not the panel appears blank or crashes the world

## Expected behavior
Panel should neither be blank nor cause world crash.

## Screenshots / Video
No screenshots taken yet. Log attached.
[DESKTOP-3U8M7V3 - 2021.1.3.619 - 2021-01-03 04_14_02.log](https://github.com/Frooxius/NeosPublic/files/5761581/DESKTOP-3U8M7V3.-.2021.1.3.619.-.2021-01-03.04_14_02.log)

## Bug information:
 - How often does it happen: Reproduced a second time with exact same problems, possibly 100% chance of occurring.
 - Neos Version: 2021.1.3.612
 - Neos Platform: Windows
 - Link to reproduction World/Item: (None in this case)
 - Regression: This has only appeared in 2021.1.3.612

## Additional context
First report, wooo

## Reporters:
Ukilop
Polaris

**Comment by dotuser** [comment]
Ukilop is on the previous build, which may be why when he added it the world crashed. We created a new world hosted by him for him and I to replicate, it crashed me but he remained on (when he tried to add the component). When I tried to add it, the panel was simply blank, as it is in a session where only I am connected. Below is the log from the replication attempt.

[DESKTOP-3U8M7V3 - 2021.1.3.619 - 2021-01-03 04_30_30.log](https://github.com/Frooxius/NeosPublic/files/5761593/DESKTOP-3U8M7V3.-.2021.1.3.619.-.2021-01-03.04_30_30.log)
